### PR TITLE
docs: clarify local trigger automation

### PR DESCRIPTION
**Pull request by zhmushan**
## Summary
- Guide the agent through the full local trigger closed-loop in the `lark-apps` skill: distinguish "configure only", "enable an existing disabled trigger", "test an existing online trigger", "handler only", "publish but do not start", and "publish then start/test" intents, so a created trigger is never reported as a running one.
- Defer business-handler details to the application project's own `trigger-guide`; `lark-apps` owns only the observable Apps/Git command lifecycle and the "code-side name must match the trigger name verbatim" constraint.
- Harden state handling: an initially disabled trigger stays fail-closed to disabled on probe success/failure/uncertain/interruption; on `+release-create` failure or an unknown result the trigger is restored or kept disabled and verified via `+release-list`; enable only after this round's release reports `finished`.
- Add `shortcuts/apps` contract tests that lock the intent routing, authorization boundaries, and fail-closed behavior.

## Test plan
- [x] `go test ./shortcuts/apps`
- [x] `go vet ./shortcuts/apps`
- [x] `gofmt -l` and `git diff --check` clean
- [x] `node scripts/skill-format-check/index.js`

## Summary by CodeRabbit

* **Documentation**
  * Expanded automation guidance for configuring, enabling, testing, publishing, starting, and troubleshooting triggers.
  * Added a complete local full-stack trigger workflow with clearer authorization and diagnostic boundaries.
  * Improved local development and release instructions, including required identity, branch, environment, and deployment sequencing.
  * Clarified optional release outputs such as online links and error details.
  * Corrected shared documentation links and updated security and database guidance.

* **Tests**
  * Added contract checks covering automation workflows, trigger boundaries, documentation links, and release guidance.

### CODE REVIEW

File: skills/lark-apps/references/lark-apps-local-dev.md [hunk 1]
```diff
@@ -31,23 +31,33 @@ npm run dev
...
+
 > `+release-create` 部署的是远端 `sprint/default` 上**已 push** 的代码，不是你本地工作区——未 commit / 未 push 的改动不会进入这次发布。所以发布前务必先把本次改动提交并推送。
 
 1. `git status` 看本次改动；`git add <本次相关文件>` 暂存后 `git commit` 提交。只提交本次任务相关的改动即可，无关的零散文件不必强求清空——发布门禁是「**本次相关改动已提交并推送**」，不是「工作区绝对干净」。
 2. `git push origin sprint/default` 把工作分支推到云端（遇非 fast-forward：先 `git pull --rebase origin sprint/default` 解决冲突再推，绝不 force-push）。
-3. `lark-cli apps +release-create --app-id <app_id>` 发起部署上线，记下返回的 `release_id`。
-4. `lark-cli apps +release-get --app-id <app_id> --release-id <release_id>` 轮询：`publishing` 继续轮询；`finished` 成功时该命令输出已含 `online_url`，直接读取它返回给用户（这是本轮发布完成后的可分享链接），无需再调 `+list`；`failed` 时该命令输出已含 `error_logs`，直接据此给出失败原因（`+list` 仅作独立查询入口）。
+3. `lark-cli apps +release-create --as user --app-id <app_id> --branch sprint/default` 发起部署上线，记下返回的 `release_id`。
+4. `lark-cli apps +release-get --as user --app-id <app_id> --release-id <release_id>` 轮询：`publishing` 时每 20 秒继续轮询，整体最多约 5 分钟；超时仍未完成时停止本轮轮询、报告 `release_id` 和当前 status。`finished` 成功时，若返回 `online_url`，可直接使用；未返回时不要编造链接。无需再调 `+list`；`failed` 时若返回非空 `error_logs`，据此给出失败原因；否则只报告 `release_id` 和当前 status，不要编造原因（`+list` 仅作独立查询入口）。
```
[the remark this answers is not in the document]
Comment (zhmushan) [reply skills/lark-apps/references/lark-apps-local-dev.md]: Done — added the creator-only visibility caveat pointing to `+access-scope-set` before `online_url` is presented as usable, in both lark-apps-local-dev.md and lark-apps-release-get.md.

**Review by liangshuo-1 — APPROVED** [review]
